lib/recipes.ts

```diff
@@ -1,19 +1,19 @@
-import fs from "fs"
-import matter from "gray-matter"
-import path from "path"
-import remark from "remark"
-import html from "remark-html"
+import fs from 'fs'
+import matter from 'gray-matter'
+import path from 'path'
+import remark from 'remark'
+import html from 'remark-html'
 
-const recipesDirectory = path.join(process.cwd(), "recipes")
+const recipesDirectory = path.join(process.cwd(), 'recipes')
 
 export function getSortedRecipesData(): RecipeData[] {
   const fileNames = getAllMDFileNamesFromDirectory(recipesDirectory)
-  const allRecipesData = fileNames.map(fileName => {
+  const allRecipesData = fileNames.map((fileName) => {
```

Comment:
: (

Reply:
God bless prettier.

> At first glance, avoiding parentheses may look like a better choice because of less visual noise. However, when Prettier removes parentheses, it becomes harder to add type annotations, extra arguments or default values as well as making other changes. Consistent use of parentheses provides a better developer experience when editing real codebases, which justifies the default value for the option.